_shared/static/css/src/_migrations/v1_v2/bootstrap.3.3.7.css

```diff
@@ -137,7 +139,9 @@ th {
 /* … */
 
 html {
- font-size:10px;
+ /* Disabled by TACC */
+ /* FAQ: Let TACC handle influence `rem`; this and more may be deleted anyway */
+ /* font-size:10px; */
```

Comment:
__Explanation__
This style has been being overwritten anyway for more than a year.